R-package/tests/helper_scripts/install_deps.R

```diff
@@ -15,10 +15,8 @@ pkgs <- c(
   "Ckmeans.1d.dp",
   "vcd",
   "testthat",
-  "lintr",
```

Comment:
Is it no longer necessary to install lintr?

Reply:
ah sorry! that was a mistake, I've added this back in in https://github.com/dmlc/xgboost/pull/8545/commits/b392fbbd464fc109fd08e069155acadc8768d248